keras_nlp/models/albert/albert_backbone.py

```diff
@@ -30,7 +31,7 @@ def albert_kernel_initializer(stddev=0.02):
     return keras.initializers.TruncatedNormal(stddev=stddev)
 
 
-@keras.utils.register_keras_serializable(package="keras_nlp")
+@keras_nlp_export("keras_nlp.models.albert.AlbertBackbone")
```

Comment:
You should use the public path (as documented on keras.io, e.g. https://keras.io/api/keras_nlp/models/albert/albert_backbone/#albertbackbone-class). Here it is `"keras_nlp.models.AlbertBackbone"`. Same for the other symbols!

Reply:
sure ! will make the change